src/calculateRentalCost.js

```diff
@@ -5,6 +5,19 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const price = 40;
```

Comment:
1) use CONSTANTS for all the static values
```suggestion
  const PRICE_PER_DAY = 40;
```
CONSTANT - This is a variable that is created only 1 time and never changes.
2) Move CONSTANTS outside the function, so that they don't recreate each time the function is called

Look at the [checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md)